internal/tsc_wrapped/tsc_wrapped.ts

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
if it's not a bin then we could drop this too?

Reply:
yup. done